src/model/window-guardian.ts

```diff
@@ -54,7 +54,10 @@ const makeWindowGuardianConfig = (
 			adUnit: config.adUnit,
 			showRelatedContent: true,
 			ajaxUrl: config.ajaxUrl,
-			hbImpl: config.hbImpl,
+			hbImpl: {
+				prebid: config.switches.prebidHeaderBidding,
+				a9: config.switches.a9HeaderBidding,
+			},
```

Comment:
Do we still need this? I think that it’s only needed in `dfpEnv` in the end.

Reply:
Good point I don't think its actually used anywhere in DCR?